contrib/babelfishpg_tsql/src/backend_parser/gram-tsql-epilogue.y.c

```diff
@@ -752,13 +752,39 @@ TsqlExpressionContains(char *colId, Node *search_expr, core_yyscan_t yyscanner)
 static Node *
 makeToTSVectorFuncCall(char *colId, core_yyscan_t yyscanner, Node *pgconfig)
 {
-    Node *col;
-    List *args;
-    Node *replaceSpecialCharsFunc;
-    List *replaceSpecialCharsArgs;
-
-    /* Create a ColumnRef node for the column */
-    col = makeColumnRef(colId, NIL, -1, yyscanner);
+    Node	*col;
+    List	*args;
+    Node	*replaceSpecialCharsFunc;
+    List	*replaceSpecialCharsArgs;
+	char 	*tableName = NULL;
+    char 	*columnName = NULL;
+	size_t	tableNameLength;
+	char 	*dot;
+
+    /* Parse the column reference for alias table */
+    dot = strchr(colId, '.');
```

Comment:
Finding the first dot may not guarantee we are using alias. Did you try out some variations of the below cases in your test?
```
select * from .dbo.t
select * from ...t
select .t.a from .t 
```
If not, please add them to the test. You can refer https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/t-sql/language-elements/transact-sql-syntax-conventions-transact-sql.md (not sure whether Babelfish supports ServerName though).